src/assets/main.css

```diff
@@ -138,13 +152,13 @@ ul.btn-group {
   flex-grow: 0.25 !important;
 }
 .control.checkbox-container {
-  flex-flow: unset;
+  flex-flow: unset;;
 }
 .control.checkbox-container label {
```

Comment:
Why isn't this rule specific enough? Why the extra class?

Reply:
I will try your suggestion. I found I needed separate styling for the radio-group labels and the checkbox labels, so that visually they look similar.

Reply:
OK, it worked, thanks!

Reply:
wondering about the checkbox-group on MIDIView, would you prefer it on 2 lines (like elsewhere in the app)? I had to make a scoped style for it....

Reply:
I think MIDI view deserves dedicated styles. That many checkboxes in a row is once-in-an-app exception.